eclair-core/src/main/scala/fr/acinq/eclair/channel/ChannelTypes.scala

```diff
@@ -257,7 +257,11 @@ case class ChannelVersion(bits: BitVector) {
 
   require(bits.size == ChannelVersion.LENGTH_BITS, "channel version takes 4 bytes")
 
-  val commitmentFormat: CommitmentFormat = DefaultCommitmentFormat
+  val commitmentFormat: CommitmentFormat = if (isSet(USE_ANCHOR_OUTPUTS_BIT)) {
```

Comment:
Should define a `def hasAnchorOutputs: Boolean` along other features

Reply:
Sounds good, I'll add that.